src/fu-engine.c

```diff
@@ -106,8 +106,9 @@ struct _FuEngine
 	gchar			*host_machine_id;
 	JcatContext		*jcat_context;
 	gboolean		 loaded;
+	gboolean		 host_security_valid;
```

Comment:
Rather than a boolean attribute, what do you think about just setting `host_security_id` to `NULL` when no longer valid and check for that everywhere?

Reply:
Something like that?